test/runtests.py

```diff
@@ -16,6 +16,7 @@
 
 from mozillapulse import consumers, publishers
 from mozillapulse.messages.test import TestMessage
+from kombu import Exchange
```

Comment:
``isort`` put ``kombu`` in the same section right before (though I had it after due to my WRONGNESS.  :)) mozillapulse because I guess that's 3rd party wrt Pulse Guardian. I'm fine with whatever you prefer.  Want it in its own space-separated section then?